utils/ANLEra_auto_working.cfg

```diff
@@ -23,7 +23,7 @@
                 side=$side_number
                 canrecruit=no
                 [filter_location]
-                    terrain="Sm^Vm" # FIXME: This terrain code does not exist
+                    terrain="Sm^Vm" # FIXME: This terrain code does exist, but the Merfolk Worker does not build a village with this code
```

Comment:
Oh, indeed, it does exist. And, with the changes I did recently, merman can build villages there, in the case that the seldom used Sm terrain already exists on the map.

Reply:
The original intent was for it to be the "university" of the merfolk

Reply:
Feel free to remove the comment =)